visualization/package.json

```diff
@@ -128,8 +128,8 @@
 		"lint-staged": "^13.0.0",
 		"nw-builder": "^3.7.4",
 		"puppeteer": "^17.0.0",
-		"remark-html": "^13.0.1",
-		"remark-loader": "^4.0.0",
+		"raw-loader": "^4.0.2",
```

Comment:
Raw loader seems to be deprecated soon.
"DEPREACTED for v5: please consider migrating to [asset modules](https://webpack.js.org/guides/asset-modules/)."

Reply:
I replace it.